### PR TITLE
Scripts to check issue 2612

### PR DESCRIPTION
**Pull request by dboltovskyi**
ATF Test Scripts to check [#2612](https://github.com/smartdevicelink/sdl_core/issues/2612)

This PR is **ready** for review.

### Summary
A test script to verify that issue is fixed

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Please do not review this PR for now since it hasn't passed our internal review yet. Thank you.

File: test_scripts/Defects/5_0/2612_Activate_non-media_app_during_embedded_audio_source.lua [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/2612
+---------------------------------------------------------------------------------------------------
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2612_Activate_non-media_app_during_embedded_audio_source.lua]: Please add the sequence description

Comment (dboltovskyi) [reply test_scripts/Defects/5_0/2612_Activate_non-media_app_during_embedded_audio_source.lua]: @GetmanetsIrina Done

**Comment by dboltovskyi** [comment]
Livio team please notice script is ready for review.

**Comment by GetmanetsIrina** [comment]
PR is rebased on actual develop(46a76abd51a39d0f89a63c9af4d98861beacadc2) branch.

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(a7db5cc) branch.
Script is applicable for all policy flows and transport types.
Script is moved to 6_1 folder.
Test set for script is added.

**Comment by dboltovskyi** [comment]
@jordynmackool Please notice PR is ready for PM review.

**Pull request merged**